ci/ci/shell_helper.py

```diff
@@ -2,4 +2,4 @@
 
 
 def shell(*args):
-    sp.run(args, capture_output=True, check=True)
+    sp.run(args, stderr=sp.PIPE, stdout=sp.PIPE, check=True)
```

Comment:
What's the reason for this change?

Reply:
Python 3.7 added capture_output. We're reverting to python 3.6.

Reply:
hmmm... that must mean pipeline will eventually need to be updated.